numpy/core/src/multiarray/conversion_utils.c

```diff
@@ -78,6 +78,29 @@ PyArray_OutputConverter(PyObject *object, PyArrayObject **address)
     }
 }
 
+static npy_intp
+intp_from_scalar(PyObject *ob)
+{
+    /*
+    * Convert the given value to an integer. Then store it and check for
+    * errors.
+    */
```

Comment:
```suggestion
     * Convert the given value to an integer. Then store it and check for
     * errors.
     */
```
style nit: keep the asterisks/stars aligned.